pages/pages.go

```diff
@@ -18,20 +19,22 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-// Resp is the response we serve for file queries.
-type Resp struct {
-	Title     string          `json:"title"` // Human-readable title.
-	Slug      string          `json:"slug"`  // URL-slug.
-	URL       string          `json:"url"`
-	UpdatedAt string          `json:"updated_at"` // Body update time.
-	Image     string          `json:"image"`      // Path/URL/Placeholder to image.
-	Message   string          `json:"message"`    // Message to show at top
-	Body      string          `json:"body"`       // Main content of the page.
-	Sidebar   string          `json:"sidebar"`    // The sidebar of the page.
-	Sort      *int            `json:"sort"`       // The order that the tab should appear in on the page
-	Expanded  bool            `json:"expanded"`   // Should the Nav-tree rooted in this node always be expanded one step when loaded?
-	Anchors   []anchor.Anchor `json:"anchors"`    // The list of anchors to headers in the body.
-	Nav       []*Node         `json:"nav,omitempty"`
+type LangLookup map[string]string
+type LangAnchorLookup map[string][]anchor.Anchor
+
+// TODO: Better type name
+type RespStore struct {
```

Comment:
I guess this represents a page? So maybe `Page` could work?

Reply:
Good idea!